datasync/chaindatafetcher/kas/contract_caller.go

```diff
@@ -93,7 +102,8 @@ func getCallOpts(blockNumber *big.Int, timeout time.Duration) (*bind.CallOpts, c
 // - the call can be reverted: returns "evm: execution reverted"
 // handleSupportsInterfaceErr handles the given error according to the above explanation.
 func handleSupportsInterfaceErr(err error) error {
-	if err != nil && (strings.Contains(err.Error(), errMsgEmptyOutput) || strings.Contains(err.Error(), errMsgEvmReverted)) {
+	if err != nil && (strings.Contains(err.Error(), errMsgEmptyOutput) || strings.Contains(err.Error(), errMsgEvmReverted) ||
+		strings.Contains(err.Error(), errMsgNoContractCode)) {
```

Comment:
Instead of define new error, please use already defined error.
```suggestion
		strings.Contains(err.Error(), bind.ErrNoCode)) {
```

Reply:
Updated on 695365ffb27a4e1b68e7f9be4284c59ebe62ed19.